vimrc

```diff
@@ -23,6 +23,33 @@ if has('nvim')
   let g:tern_request_timeout = 1
   let g:tern_show_signature_in_pum = 0
   set completeopt-=preview
+
+  " == neomake/neomake ==
+  let g:neomake_javascript_jshint_maker = {
```

Comment:
that is not true, `g:neomake_javascript_eslint_maker` is used to override the pre-existing eslint maker